### PR TITLE
Issue 1329

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://github.com/openstax/unified/issues/1329 

TODO:
- [x] Icon for Practice is not ready yet - [slack conversation](https://openstax.slack.com/archives/C8V1XET1R/p1602773276497000)

### CODE REVIEW

File: src/app/content/components/Toolbar/PracticeQuestionsButton.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { PlainButton, toolbarDefaultButton, toolbarDefaultText } from './styled';
+
+// tslint:disable-next-line:variable-name
+export const PracticeQuestionsWrapper = styled(PlainButton)`
+  ${toolbarDefaultButton}
+`;
+
+// tslint:disable-next-line:variable-name
+const PracticeQuestionsIcon = styled.img`
+  padding: 0.2rem;
```
Comment (KarinaMendez2) [inline src/app/content/components/Toolbar/PracticeQuestionsButton.tsx:16]: why not ` ${toolbarIconStyles}`?

File: src/app/content/components/Toolbar/PracticeQuestions.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      </Services.Provider>
+    </Provider>);
+
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+
+  it('clicking button works', () => {
+    store.dispatch(receiveFeatureFlags([practiceQuestionsFeatureFlag]));
+
+    const component = renderer.create(<Provider store={store}>
```
Comment (KarinaMendez2) [inline src/app/content/components/Toolbar/PracticeQuestions.spec.tsx:53]: what about putting all renderer.create into a render() function?

**Review by KarinaMendez2 — CHANGES_REQUESTED** [review]
couple comments, everything else looks good

File: src/app/content/components/Toolbar/PracticeQuestionsButton.tsx (hunk 1 above)
Comment (PiotrKozlowski) [reply src/app/content/components/Toolbar/PracticeQuestionsButton.tsx:16]: icon looks bigger than in the design with these styles

File: src/app/content/components/Toolbar/PracticeQuestions.spec.tsx (hunk 2 above)
Comment (PiotrKozlowski) [reply src/app/content/components/Toolbar/PracticeQuestions.spec.tsx]: I think this is a good idea if all of the tests use the same JSX

**Comment by PiotrKozlowski** [comment]
@KarinaMendez2 I've responded to your comments and also pushed a commit that will make sure we are removing the print button and changing styles of my highlights button only if practice questions are enabled
https://openstax.slack.com/archives/C8V1XET1R/p1603194876087500

**Review by KarinaMendez2 — APPROVED** [review]

**Pull request merged**